docs/design.md

```diff
@@ -1,317 +1,136 @@
 # Design of the Service Catalog
 
-Table of Contents
+This document covers the architecture and design of Service Catalog.
+
+#### Table of Contents
+
 - [Overview](#overview)
 - [Terminology](#terminology)
 - [Open Service Broker API](#open-service-broker-api)
 - [Service Catalog Design](#service-catalog-design)
-- [Current Design](#current-design)
 
 ## Overview
 
-The Service Catalog is an implementation of the
-[Open Service Broker API](https://github.com/openservicebrokerapi) for
-Kubernetes. It allows for:
-- a Service Broker to register with Kubernetes
-- a Service Broker to specify the set of Services (and variantions of those
-  Services) to Kubernetes that should then be made available to Kubernetes'
-  users
-- a user of Kubernetes to discover the Services that are available for use
-- a user of Kubernetes to request for a new ServiceInstance of a Service
-- a user of Kubernetes to link an ServiceInstance of a Service to a set of Pods
+The Service Catalog is an client of the 
+[Open Service Broker API](https://github.com/openservicebrokerapi) (OSB API)
+that translates the Kubernetes resource model into OSB API calls to a 
+service broker.
+
+It has the following high level features:
+
+- Service Catalog users can register service brokers with Kubernetes
+- Service Catalog can fetch the services and plans (called the catalog) from 
+each service broker, and make it available to Kubernetes users
+- Kubernetes users can request a new service and plan by submitting a 
```

Comment:
are we assuming the reader is familiar with service (class?) and plan from OSBAPI?